### DISCUSSION TITLE
Is is possible to use tradicional tremolos?

### DISCUSSION THREAD

**Discussion by charlesneimog** [DigiScore/neoscore#86] [Q&A] [answered]
Is is possible to use tradicional tremolos?

![image](https://user-images.githubusercontent.com/31707161/209241307-ed853048-c79c-401d-b40d-8660f31c6fd1.png)

**Comment by ajyoon** [comment, marked as answer]
Hmm.. we don't have automatic support for this, but it's certainly *possible*. I just threw together this hacky manual placement of a tremolo, though I didn't have time to work out the alignment (whole note positioning is kind of weird since it doesn't have the usual stem attachment point). Robust support for this would be a reasonable thing to include directly in Chordrest for sure.

![tremolo](https://user-images.githubusercontent.com/15976214/209244699-757424d5-8d0e-41a0-b904-83b8c7e27e35.png)

```py
from neoscore.common import *

neoscore.setup()

staff = Staff((Mm(10), Mm(10)), None, Mm(150))
clef = Clef(ZERO, staff, "treble")

c = Chordrest(Mm(10), staff, ["a"], (1, 1))
target_notehead = c.highest_notehead
MusicText(
    (ZERO, staff.unit(-2)),
    target_notehead,
    "tremolo3",
)

neoscore.show()

```

I'm explicitly grabbing glyphs from SMuFL's tremolo section which you can see more on here https://w3c.github.io/smufl/latest/tables/tremolos.html

**Reply by charlesneimog** [reply]
Great, thank you very much for the help!
